Automattic/wp-calypso
# Move theme upgrade modal

Related to #81089

## Proposed Changes

Move the onboarding specific `UpgradeModal` to a central calypso component `ThemeUpgradeModal` so that it can be used in other theme UIs

## Testing Instructions



1. Use calypso live
2. Add D118900-code to sandbox and sandbox (so you can take upcoming marketplace changes)
3. Go to /start, choose a free theme and first preview then 'Unlock' Premium, Marketplace and Woo themes - they should work exactly as they do on live.

## Pre-merge Checklist
